internal/backend/db/db.go

```diff
@@ -127,7 +127,7 @@ type DB interface {
 	UpdateIntegration(ctx context.Context, i sdktypes.Integration) error
 	DeleteIntegration(ctx context.Context, id sdktypes.IntegrationID) error
 	GetIntegration(ctx context.Context, id sdktypes.IntegrationID) (sdktypes.Integration, error)
-	ListIntegrations(ctx context.Context, filter sdkservices.ListIntegrationsFilter) ([]sdktypes.Integration, error)
+	ListIntegrations(ctx context.Context) ([]sdktypes.Integration, error)
```

Comment:
why did you remove the filter ?

Reply:
wasn't really used. tbh i think we need to remove all the integrations support from the database - it is not used rn as the integrations are hardcoded.